tools/docker/Dockerfile

```diff
@@ -44,6 +48,16 @@ RUN apt-get -qqy install \
     libindicator3-7 \
     libindicator7
 
+# To speed up runs of webkitgtk_minibrowser nightly we install all the
```

Comment:
How much size does this add to the docker image, and how much time does it save?

Since this would only take effect when updating the image, which we don't do often, will the dependencies not become increasingly out of sync and need to be updated at runtime?

Reply:
It makes the docker image 1GB bigger (from 1.27GB to 2.2GB). It saves around 6-10 minutes:
 * 3-7 minutes of download (depends on the download speed of the ubuntu apt repositories)
 * ~3 minutes of installing packages

Reply:
Given that WebKitGTK is still in a bit of an experimental phase that sounds like it's adding a bit too much burden on the other uses of the same image and could increase total transfer size, but I'll let @jgraham weigh in on that.

Reply:
Yeah, I think I'd prefer to make the WebKitGtk runs bear that cost for now. If we think that adding 10 minutes per job is too much I guess we could have a second Docker image here which inherits from the base one and just adds the extra bits to do this update (yes, that undoes part of the motivation for this PR). That adds the maintainance burden of requiring us to update both dockerfiles when we change the base image.

Reply:
@clopez I see you've rebased now but these changes remain. Please ping when it's ready for final review.